nautobot_ssot/__init__.py

```diff
@@ -21,7 +21,7 @@ class NautobotSSOTPluginConfig(PluginConfig):
     description = "Nautobot Single Source of Truth."
     base_url = "ssot"
     required_settings = []
-    min_version = "1.0.3"
+    min_version = "1.4.0"
```

Comment:
I think we should document the change on the supported version on a compatibility matrix

Reply:
Fixed by updating docs

Reply:
Thanks @snaselj ,but because this plugin is widely used, we should be more explicit with a table like this

| nautobot-plugin-ssot version | min Nautobot version |
| < 1.2.0 |  1.0.3 |
| 1.3.x | 1.4.0 |

Reply:
I see, this should be part of another PR:
https://github.com/nautobot/nautobot-plugin-ssot/pull/80